app/components/projects/wizard/help_text_component.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= component_wrapper do %>
+  <% if attribute_help_text.present? %>
+    <%= render Primer::Beta::Text.new(tag: :p, color: :muted) do %>
+      <%= render Primer::Beta::Octicon.new(icon: :info) %>
+      <%= attribute_help_text.attribute_caption %>
+    <% end %>
```

Comment:
According to the visuals, only the icon should be grey, not the text. Further, there is some spacing missing
  

```suggestion
    <%= render Primer::Beta::Text.new(tag: :p) do %>
      <%= render Primer::Beta::Octicon.new(icon: :info, color: :muted, mr: 2) %>
      <%= attribute_help_text.attribute_caption %>
    <% end %>
```